clinica/iotools/bids_utils.py

```diff
@@ -818,3 +818,8 @@ def run_dcm2niix(
             ),
             lvl="warning",
         )
+
+
+def write_to_tsv(dataframe: DataFrame, buffer: Union[PathLike, BinaryIO]) -> None:
```

Comment:
This line should be something like this:
```suggestion
def write_to_tsv(dataframe: pd.DataFrame, buffer: Union[PathLike, BinaryIO]) -> None:
```

Reply:
Also need to add some lacking imports for the `PathLike` and `BinaryIO` type annotations, something like this:
```
from os import PathLike
from typing import List, Optional, Union, BinaryIO
```
at the beginning of the file.

Reply:
I have only imported DataFrame from pandas not to import the whole library. Is that okay ?

Reply:
> I have only imported DataFrame from pandas not to import the whole library. Is that okay ?

Yes, that's the right way to do it! Please run the `isort` hook before committing, it seems that the order of the imports is not the right one.

Reply:
> Please run the isort hook before committing

or `make format`

Reply:
the `isort`hook has run the imports should be in the correct order